### PR TITLE
fix typo

### PR DESCRIPTION
**Pull request by yuxiaobo96**
Signed-off-by: yuxiaobo <yuxiaobogo@163.com>

### Ⅰ. Describe what this PR did
Improve document quality.

### Ⅱ. Does this pull request fix one issue?

NONE

### Ⅲ. Why don't you add test cases (unit test/integration test)? (你真的觉得不需要加测试吗？)
Don't need.

### CODE REVIEW

File: supernode/daemon/util/filter.go [hunk 1]
```diff
@@ -28,19 +28,19 @@ import (
 )
 
 const (
-	// PAGENUM identity the page number of the data.
+	// PAGENUM identities the page number of the data.
```
Comment (yeya24) [inline supernode/daemon/util/filter.go]: I think `identifies` is much appropriate

File: supernode/daemon/util/filter.go [hunk 2]
```diff
@@ -28,19 +28,19 @@ import (
...
 	// The default value is 0.
 	PAGENUM = "pageNum"
 
-	// PAGESIZE identity the page size of the data.
+	// PAGESIZE identities the page size of the data.
```
Comment (yeya24) [inline supernode/daemon/util/filter.go]: ditto

File: supernode/daemon/util/filter.go [hunk 3]
```diff
@@ -28,19 +28,19 @@ import (
...
 	// If this value equals 0, return all values.
 	PAGESIZE = "pageSize"
 
-	// SORTKEY identity the sort key of the data.
+	// SORTKEY identities the sort key of the data.
```
Comment (yeya24) [inline supernode/daemon/util/filter.go]: ditto

File: supernode/daemon/util/filter.go [hunk 4]
```diff
@@ -28,19 +28,19 @@ import (
...
 	// Each mgr needs to define acceptable values based on its own implementation.
 	SORTKEY = "sortKey"
 
-	// SORTDIRECT identity the sort direct of the data.
+	// SORTDIRECT identities the sort direct of the data.
```
Comment (yeya24) [inline supernode/daemon/util/filter.go]: ditto

File: supernode/daemon/util/filter.go (hunk 1 above)
Comment (yuxiaobo96) [reply supernode/daemon/util/filter.go]: As you can see, I have already modified it.
But I can't understand what you mean,“I think identifies is much appropriate.”

Comment (yeya24) [reply supernode/daemon/util/filter.go]: identity is a verb, but identity is a noun.

Comment (yuxiaobo96) [reply supernode/daemon/util/filter.go]: Wow, you are great.
Thank you.

**Comment by yeya24** [comment]
Please squash your commits. @yuxiaobo96

**Comment by yuxiaobo96** [comment]
Ok, I have squashed.

**Comment by Starnop** [comment]
LGTM.

**Pull request merged**